### DISCUSSION TITLE
max_concurrent

### DISCUSSION THREAD

**Discussion by Nebemires** [swar/Swar-Chia-Plot-Manager#352] [Q&A]
Hello!
Problem is that i cant get working max_concurrent: 12.
So i changed  from threads: 2 to  threads: 1 and  from max_concurrent: 10 to   max_concurrent: 12, max_for_phase_1:7, stagger_minutes: 30 and settings does no changed. Restarting manager does not helped. Any ideas what else i can try ?
Windows 10 Pro

**Comment by swar** [comment]
Please use Github Discussions next time. This is not an Issue with the software.

**Comment by Kaveman42** [comment]
Make sure your global max is set to 12 as well. It should be around line 84 of the config file. If that is update, maybe post your config file here to see what setting is preventing max concurrent. 

```
`global:
  # These are the settings that will be used globally by the plot manager.
  #
  # max_concurrent: The maximum number of plots that your system can run. The manager will not kick off more than this
  #                 number of plots total over time.
  max_concurrent: 10`
```
